docker/lite/Dockerfile

```diff
@@ -36,6 +36,16 @@ RUN make install PREFIX=/vt/install
 # Start over and build the final image.
 FROM debian:bullseye-slim
 
+# Install mysqlbinglog
+RUN apt-get update && apt-get -y install libssl-dev gnupg
```

Comment:
@frouioui Why is this installing `libssl-dev` instead of `libssl1.1` (which is the one for bookworm). We don't need the headers here and that might pull in more things accidentally?

Reply:
Fixed!